index.js

```diff
@@ -36,12 +38,25 @@ extendConfig(function (config, userConfig) {
     validate(conf, 'path', 'string');
     validate(conf, 'runOnCompile', 'boolean');
     validate(conf, 'clear', 'boolean');
-    validate(conf, 'flat', 'boolean');
```

Comment:
Do we want to re-add this boolean validation then?

Reply:
Yes

Reply:
Done